libexec/jenv-remove

```diff
@@ -9,16 +9,19 @@ if [ "$1" = "--complete" ]; then
   exec jenv-versions --bare
 fi
 
-JENV_VERSION="$1" 
-                  
-                            
-if [ -L "$JENV_ROOT/versions/$JENV_VERSION" ];
-then         
-	rm -f $JENV_ROOT/versions/$JENV_VERSION
-	rm -f ${JENV_ROOT}/$JENV_VERSION.time
-  $(jenv-rehash)             
-   echo "JDK $JENV_VERSION removed"
 
-else
-	echo "$JENV_VERSION is not a managed version of Java "
-fi
+for jenv_version in "$@"
+do
+  JENV_VERSION=${jenv_version}
+
+  if [ -L "$JENV_ROOT/versions/$JENV_VERSION" ];
+  then
+    rm -f $JENV_ROOT/versions/$JENV_VERSION
+    rm -f ${JENV_ROOT}/$JENV_VERSION.time
+    $(jenv-rehash)
```

Comment:
I'd suggest calling `jenv-rehash` after the loop.

Reply:
Indeed, it will improve the performance

Reply:
Hello @emcrisostomo, thanks for the contribution and sorry for the delay to review this PR.

Could you apply the suggestion in the comment above or enable the _Allow edits from maintainers_ option?

Thanks

Reply:
Thanks @kevinleturc, no worries, thanks.
@voronaam is right, no need to rehash at every loop cycle. I've updated the PR to reflect the change, and squashed the commits.